src/Network/Network/Models/PSApplicationGateway.cs

```diff
@@ -83,6 +85,36 @@ public class PSApplicationGateway : PSTopLevelResource
         [Ps1Xml(Target = ViewControl.Table)]
         public bool? EnableFips { get; set; }
 
+        [Ps1Xml(Target = ViewControl.Table)]
+        public bool? EnableRequestBuffering
+        {
+            get
+            {
+                return this.GlobalConfiguration.EnableRequestBuffering;
```

Comment:
what is GlobalConfiguration is null

Reply:
Thanks for the catch... I had just been considering new AppGws where the Global Config was set. I forgot that we have existing AppGws where Global Config would not have been initialized. I've added a null check in the getters where we'll now initialize Global Config if it doesn't already exist.